src/matter/mdns/MdnsBroadcaster.ts

```diff
@@ -30,6 +30,8 @@ export class MdnsBroadcaster implements Broadcaster {
     ) {}
 
     setCommissionMode(mode: number,deviceName: string, deviceType: number, vendorId: VendorId, productId: number, discriminator: number) {
+        logger.debug(`announce commissioning mode ${mode} ${deviceName} ${deviceType} ${vendorId} ${productId} ${discriminator}`);
```

Comment:
nit: Announce

This method actual doesn't announce but just set the mode. The announce() method should be called to announce...